tests/test_modeling_common.py

```diff
@@ -1284,3 +1288,9 @@ def test_model_from_pretrained(self):
             model = BertModel.from_pretrained(model_name, output_attentions=True, output_hidden_states=True)
             self.assertEqual(model.config.output_hidden_states, True)
             self.assertEqual(model.config, config)
+
+    def test_model_from_pretrained_with_different_pretrained_model_name(self):
+        model = BertModel.from_pretrained(TINY_BERT)
+        self.assertIsNotNone(model)
+
+        self.assertRaises(AssertionError, BertModel.from_pretrained, TINY_T5)
```

Comment:
```suggestion
        with self.assertRaises(Exception) as context:
            BertModel.from_pretrained(TINY_T5)
        self.assertTrue("You tried to initiate a model of type" in str(context.exception))
```

Let's check the actual assert message here, just in case it asserts on something else and then this test would be misleading.

Just please test that it works. thank you.